app/assets/javascripts/admin/index_utils/services/pending_changes.js.coffee

```diff
@@ -6,7 +6,7 @@ angular.module("admin.indexUtils").factory "pendingChanges", ($q, resources, Sta
     add: (id, attr, change) =>
       @pendingChanges["#{id}"] = {} unless @pendingChanges.hasOwnProperty("#{id}")
       @pendingChanges["#{id}"]["#{attr}"] = change
-      StatusMessage.display('notice', "You have made #{@changeCount(@pendingChanges)} unsaved changes")
+      StatusMessage.display('notice', "You have unsaved changes")
```

Comment:
I know that I'm asking for an unrelated thing here... but if we could take advantage of this change to create a i18n key and use `t()` instead of a literal string, that would be awesome. But only if it takes really small added effort.

In any case, many thanks for your contribution @HugsDaniel 🎉